### ISSUE TITLE
Feature request: connect to existing jupyterlab server using `jlab` command

### ISSUE THREAD

**Issue by lukelbd** [jupyterlab/jupyterlab-desktop#629]
Recently I noticed support for connecting to existing servers was added (#15), which is awesome. However it seems the `jlab` command does not recognize arguments that are URLs pointing to servers. Could this be added in a future release?

Example: Currently, the following will create a new session and open the specified notebook:

```sh
jlab notebook.ipynb
```

However, this simply opens a new window on the landing page instead of connecting to the server:

```sh
jlab http://127.0.0.1:2001/lab?token=xxx
```

**Comment by lukelbd** [comment]
Any updates? Looks like it would require editing `createFromArgs` in `sessonConfig.ts`:

https://github.com/jupyterlab/jupyterlab-desktop/blob/ca030571a8478a157c5bd688fed3d807958f6970/src/main/config/sessionconfig.ts#L106-L171

Seems simple enough but not sure of the commands needed to create + connect to a session from a URL.

**Comment by mbektas** [comment]
hi @lukelbd , this is a good idea. I will look into adding this feature.

**Comment by mbektas** [comment]
@lukelbd this feature is now available with v4.0.7-1. Please give it a try and create a new issue if you run into problems. Here is an [example](https://github.com/jupyterlab/jupyterlab-desktop#jlab-command-line-launch-examples).
